azure-pipelines.yml

```diff
@@ -19,6 +19,9 @@ jobs:
         publishJUnitResults: true
         testResultsFiles: '**/TEST-*.xml'
 
+- job: other_tests
```

Comment:
From reading the Azure pipeline docs, it is very unclear to me, if those jobs will run in parallel. We have the following setting in Azure:
![image](https://user-images.githubusercontent.com/5088104/154927427-af56bc8c-3d0e-4964-9a8e-a61b43cb28d3.png)

From the start timestamps, it looks like they ran sequentially (with `other_tests` running first).

Reply:
they will, if we have multiple agents, but one agent will always run one job.

Also, another upside of having multiple jobs is because we can rerun failed jobs only